src/pages/GroupPage/Management/CreateQuestion/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react'
+import { BiError, BiPlus } from 'react-icons/bi'
+
+import {
+  Box,
+  Button,
+  Flex,
+  Text,
+  Textarea,
+  useDisclosure,
+} from '@chakra-ui/react'
+import { useMutation } from '@tanstack/react-query'
+
+import {
+  CreateGroupQuestionPayload,
+  createGroupQuestion,
+} from '@/api/services/question/group.api'
+import { AlertModal } from '@/components/Modal/AlertModal'
+import { FormModal } from '@/components/Modal/FormModal'
+
+interface GroupQuestionCreateModalProps {
+  isOpen: boolean
+  onClose: () => void
+  groupId: number
+}
+
+export const GroupQuestionCreateModal = ({
+  isOpen,
+  onClose,
+  groupId,
+}: GroupQuestionCreateModalProps) => {
+  const [questionContent, setQuestionContent] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const errorAlert = useDisclosure()
+
+  const { mutate: createQuestion } = useMutation({
+    mutationFn: (payload: CreateGroupQuestionPayload) =>
+      createGroupQuestion(payload),
+    onSuccess: () => {
+      onClose()
+      setQuestionContent('')
+    },
+    onError: () => {
+      setErrorMessage('질문 생성에 실패하였습니다')
+      errorAlert.onOpen()
+    },
+  })
+
+  const handleCreateQuestion = () => {
+    if (!questionContent.trim()) {
+      setErrorMessage('질문을 입력해주세요')
```

Comment:
errorAlert 모달이 팝업될 때는 질문 추가 모달은 닫는 건 어떤가요?

```suggestion
      setErrorMessage('질문을 입력해주세요')
      onClose()
```

Reply:
오옹 닫도록 수정하겠습니다!!